### PR TITLE
94 - Bump GCT version. Use Structured Outputs/JSON Mode

### PR DESCRIPTION
**Pull request by ctmartinez1992**

### CODE REVIEW

**Review by FilipaSdS — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -144,6 +144,8 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.env.test
```
Comment (kenvontucky) [inline .gitignore:147]: this is already present a few lines above

File: labs/api/codemonkey_endpoints.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from fastapi import APIRouter, HTTPException
 from labs.api.types import CallLLMRequest, CodeMonkeyRequest, GithubModel
-from labs.config import LITELLM_API_KEY
+from labs.config.settings import LITELLM_API_KEY
```
Comment (kenvontucky) [inline labs/api/codemonkey_endpoints.py:3]: the import for env variables must be `from labs.config import settings` and then use `settings.LITELLM_API_KEY`

File: labs/middleware.py [hunk 3]
```diff
@@ -67,32 +49,85 @@ def call_llm_with_context(github: GithubModel, issue_summary, litellm_api_key):
...
         }
     )
-    try:
-        llm_response = litellm_requests.completion_without_proxy(
-            prepared_context,
-            model=LLM_MODEL_NAME,
+    return prepared_context
+
+
+def validate_llm_response(llm_response):
```
Comment (kenvontucky) [inline labs/middleware.py:55]: lets improve this function, perhaps use a dict with all the possible cases mapped out.

**Review by kenvontucky — CHANGES_REQUESTED** [review]

File: labs/middleware.py (hunk 3 above)
Comment (ctmartinez1992) [reply labs/middleware.py]: Changed it. Check if it's good now.

**Pull request merged**